#Scenes/SplashScene.tscn

```diff
@@ -0,0 +1,42 @@
+[gd_scene load_steps=4 format=3 uid="uid://0uuiv6n1s26t"]
+
+[ext_resource type="Texture2D" uid="uid://b5yppwmgeuuat" path="res://Art/Menus/loading_splash_art-1.png" id="1_7smrv"]
+[ext_resource type="Script" path="res://#Scenes/SceneScripts/SplashScene.gd" id="1_pnbb4"]
+[ext_resource type="AudioStream" uid="uid://c0l2ntm2m8esq" path="res://Audio/fauna_uuu_test.mp3" id="3_uytcb"]
+
+[node name="SplashScene" type="Control" node_paths=PackedStringArray("splash_image", "color_rect", "uuuu_sfx")]
+layout_mode = 3
+anchors_preset = 0
+offset_right = 1920.0
+offset_bottom = 1080.0
+script = ExtResource("1_pnbb4")
+splash_image = NodePath("TextureRect")
+color_rect = NodePath("ColorRect")
+uuuu_sfx = NodePath("AudioStreamPlayer2D")
+
+[node name="ColorRect" type="ColorRect" parent="."]
+layout_mode = 0
+offset_right = 1920.0
+offset_bottom = 1080.0
+color = Color(0, 0, 0, 1)
+
+[node name="TextureRect" type="TextureRect" parent="."]
+layout_mode = 0
+offset_left = -709.0
+offset_top = -1581.0
+offset_right = 709.0
+scale = Vector2(0.682, 0.682)
+pivot_offset = Vector2(709, 1581)
+texture = ExtResource("1_7smrv")
+
+[node name="AudioStreamPlayer2D" type="AudioStreamPlayer2D" parent="."]
+stream = ExtResource("3_uytcb")
+volume_db = 3.043
+
+[node name="TextureRect2" type="TextureRect" parent="."]
```

Comment:
Just a cleanup question, is this second TextureRect used somewhere? I see it is set to invisible and not used in the code so maybe it could be removed 
If it requires approval again we can just ignore it and I'll approve the PR